app/src/main/java/de/westnordost/streetcomplete/data/osmnotes/Note.kt

```diff
@@ -35,3 +35,11 @@ data class NoteComment(
         OPENED, COMMENTED, CLOSED, REOPENED, HIDDEN
     }
 }
+
+@Serializable
+data class NoteGPXTrack(
```

Comment:
```suggestion
data class Trackpoint(
```

Also, a trackpoint per se has nothing to do with `Note`s, so it shouldn't be in the same file. Actually, it shouldn't even be in the same package. It should probably be in `data/osmtracks` or similar.

Reply:
Another thing to keep in mind after renaming: For acronyms in PascalCase or camelCase, please follow the convention we used elsewhere: e.g. `OsmApi` or `osmApi`, not `osmAPI` or `OSMApi`.